var/spack/repos/builtin/packages/py-yt/package.py

```diff
@@ -65,7 +66,12 @@ class PyYt(PythonPackage):
     depends_on("py-sympy", type=('build', 'run'))
     depends_on("python @2.7:2.999,3.4:")
 
-    @run_after('install')
```

Comment:
@citibeth I need to have a better look at `setup.py` phases, but I tend to agree with you: `develop` and `install` appear to be mutually exclusive, and so an approach like #2692 seems preferable to me.